packages/plugins/geth/src/devtxs.ts

```diff
@@ -15,20 +15,23 @@ export default class DevTxs {
     this.logger = embark.logger;
     this.events = embark.events;
   }
+
   public async init() {
-    const { rpcHost, rpcPort, wsRPC, wsHost, wsPort } = this.blockchainConfig;
-    if (wsRPC) {
-      this.web3 = new Web3('ws://' + wsHost + ':' + wsPort);
-    } else {
-      this.web3 = new Web3(rpcHost + ':' + rpcPort);
+    if (!this.shouldStartDevTxs()) {
+      return;
     }
+    this.web3 = new Web3(this.blockchainConfig.endpoint);
```

Comment:
Like @emizzle pointed out, it,s better to use the `provider` request because it does some nice touch ups (for WS for example)